.github/ISSUE_TEMPLATE/feature_addition_request.yaml

```diff
@@ -9,7 +9,7 @@ body:
 
 
         Be aware, though: most feature requests are not accepted, even if they're suggested by
-        a full-time Guava team member. [Feedback](http://stackoverflow.com/a/4543114/869736) from
+        a full-time Guava team member. [Feedback](https://stackoverflow.com/a/4543114) from
```

Comment:
I copied all of these directly from the [wiki](https://github.com/google/guava/wiki/HowToContribute), but yeah all of these should probably be https if possible. I'll update all of the links in the wiki as well.